routes/auth.js

```diff
@@ -0,0 +1,90 @@
+//Third party packeges
+const express = require('express');
+const router = express.Router();
+const bcrypt = require('Bcryptjs');
+const jwt = require('jsonwebtoken');
+
+//Local imports
+const User = require('../models/user')
+
+router.post('/register', async (req, res, next) => {
+    try{
+    const {
+        username,
+        password,
+        email
+    } = req.body
+
+    let user = await User.findOne({ email })
+    if (user) {
+        return res.send({ registerError: "email allready exsisting" })
```

Comment:
"registerError" to "message" and add "error: true"
And then in front end check if error, then do the error logic, do it to all the errors